rbac/server/api/users.py

```diff
@@ -494,6 +771,24 @@
 
 
 @USERS_BP.get("api/users/<next_id>/proposals/rejected")
+@doc.summary("Get confirmed proposals for a user, by their next_id.")
+@doc.description("Get confirmed proposals for a user, by their next_id.")
+@doc.consumes({"head": str}, location="query")
+@doc.consumes({"start": int}, location="query")
+@doc.consumes({"limit": int}, location="query")
+@doc.produces(
+    [{"id": str, "type": str, "object": str, "target": str, "approvers": [str]}],
+    content_type="Application/json",
+    description="List of confirmed proposals the given user has opened.",
+)
+@doc.response(
```

Comment:
I don't see this 400 error being thrown in the reject proposals API endpoint. Can you point it where it gets thrown in the code?